core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -253,6 +253,19 @@ private SqlLibraryOperators() {
               OperandTypes.STRING),
           SqlFunctionCategory.STRING);
 
+  /** The "STRCMP(str, str)" function that compare strings.
+   * For example, "STRCMP('a', 'b')" returns -1. */
+  @LibraryOperator(libraries = {MYSQL, POSTGRESQL, ORACLE})
+  public static final SqlFunction STRCMP =
+      new SqlFunction("STRCMP",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.cascade(ReturnTypes.explicit(SqlTypeName.INTEGER),
+              SqlTypeTransforms.TO_NULLABLE),
+          null,
+          OperandTypes.repeat(SqlOperandCountRanges.of(2),
+              OperandTypes.STRING_STRING),
+          SqlFunctionCategory.STRING);
```

Comment:
Only `OperandTypes.STRING_STRING` is okey if you have exactly 2 `CHARACTER` arguments.

Reply:
yes,It can